File: rust/src/models/boolean_effective_entitlement_value.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
pub struct BooleanEffectiveEntitlementValue {
    pub enabled: bool,
}

impl BooleanEffectiveEntitlementValue {
    pub fn new(enabled: bool) -> Self {
        Self { enabled }
    }
}

File: browser/src/models/booleanEffectiveEntitlementValue.ts
// this file is @generated

export interface BooleanEffectiveEntitlementValue {
  enabled: boolean;
}

export const BooleanEffectiveEntitlementValueSerializer = {
  _fromJsonObject(object: any): BooleanEffectiveEntitlementValue {
    return {
      enabled: object["enabled"],
    };
  },

  _toJsonObject(self: BooleanEffectiveEntitlementValue): any {
    return {
      enabled: self.enabled,
    };
  },
};
